hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -265,8 +265,8 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
 
   updateDisplayLimit(e): void {
     let res = -1;
-    if (e.key === 'all') {
-      res = Number.MAX_VALUE;
+    if (e.key === '30') {
+      res = Number.parseInt(e.key, 10);
     } else {
       res = Number.parseInt(e.key, 10);
     }
```

Comment:
I don't understand this. With the changes in both cases (if `e.key` is `'all'` and if it's `'30'`) we will set `res = Number.parseInt(e.key, 10);`

Reply:
@dombizita 

In Old design earlier It was  All in dropdown as now we are setting with Max limit=30
subpaths.length > 0 && limit === MAX_DISPLAY_LIMIT